partner_firstname/i18n/fr.po

```diff
@@ -36,13 +36,13 @@ msgstr "Est une société?"
 #. module: partner_firstname
 #: field:res.partner,lastname:0
 msgid "Last name"
-msgstr ""
+msgstr "Nom de famille"
 
 #. module: partner_firstname
 #: code:addons/partner_firstname/exceptions.py:23
 #, python-format
 msgid "No name is set."
-msgstr ""
+msgstr "Pas de nom défini"
```

Comment:
To translate it exactly it misses the dot at the end of sentence or is there a reason to remove it?